frontend/public/locales/en/data-unavailable.json

```diff
@@ -3,6 +3,17 @@
   "breadcrumbs": {
     "unable-to-continue": "Unable to continue"
   },
-  "service-unavailable": "This service is not available to you. We did not find any record for your account.",
+  "service-eligible": "You are currently not a participant in the Canadian Dental Care Plan, to find out if you are eligible please visit ",
+  "do-you-qualify.text": "Do you qualify - Canada.ca",
+  "do-you-qualify.href": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan/qualify.html",
+  "more-information": "For more information on how to apply, please visit ",
+  "how-to-apply.text": "How to apply - Canada.ca",
+  "how-to-apply.href": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html",
+  "other-enquiry": "For other enquiries, please visit the Contact us page to find out how to ",
+  "contact-us.text": "Contact Us",
+  "contact-us.href": "{{baseUri}}/en/contact-us",   
+  "service-delay": "If you have recently applied, please note that there could be a delay in your application appearing within the My Service Canada Account.  For any enquiries on the status of your application please visit \"link to online status checker\"",
```

Comment:
Is this missing a trailing period (`.`)?

Reply:
fixed